src/hooks/useEnsureCurrentGroup/index.js

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react'
+import fetchGroup from 'store/actions/fetchGroupDetails'
+import { useSelector, useDispatch } from 'react-redux'
+import presentGroup from 'store/presenters/presentGroup'
+import getGroupForCurrentRoute from 'store/selectors/getGroupForCurrentRoute'
+import { useRouter } from 'hooks/useRouter'
+import { createSelector } from 'reselect'
+import isPendingFor from 'store/selectors/isPendingFor'
+import { FETCH_GROUP_DETAILS } from 'store/constants'
+
+const selectAndPresentGroup = createSelector(
```

Comment:
Added an extra reselect here to ensure `presentGroup` wasn't being re-run on every pass through of a component using this hook, and thus possibly forcing unnecessary re-rendering if the result of this hook was passed down to a child component